docs/endpoints/openrtb2/auction.md

```diff
@@ -139,10 +139,17 @@ to set these params on the response at `response.seatbid[i].bid[j].ext.prebid.ta
 
 ```
 {
-  "pricegraularity": "One of ['low', 'med', 'high', 'auto', 'dense']", // Required property.
+  "pricegraularity": [{
+    "precision": 2,
+    "min":0.0,
+    "max":20.0
+    "increment":0.1 }], // Required property.
```

Comment:
You're not requiring this anymore, are you? I saw in the code that it defaults to `medium`.

Since we're calling those strings "deprecated" now, it seems useful to list out the default explicitly here.

Reply:
We always defaulted to medium, so actually was never required. I'll get rid of "required"